.github/workflows/run-tests.yml

```diff
@@ -83,14 +87,21 @@ jobs:
         if: inputs.os == 'ubuntu-latest'
         run: docker info
 
-      - name: Build with packages
+      - name: Download built nugets
         if: ${{ inputs.requiresNugets }}
-        env:
-          CI: false
-        run: ${{ env.BUILD_SCRIPT }} -restore -build -ci -pack -testnobuild /bl
+        uses: actions/download-artifact@cc203385981b70ca67e1cc392babf9cc229d5806 # v4.1.9
```

Comment:
Just curious, any reason why some of the actions we reference use tag names (like setup-dotnet above) but some others use direct hashes?

Reply:
@danmoseley could answer that